external-crates/move/crates/move-cli/src/base/new.rs

```diff
@@ -1,18 +1,12 @@
 // Copyright (c) The Move Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use anyhow::anyhow;
+use anyhow::{self, bail};
 use clap::*;
 use move_core_types::identifier::Identifier;
 use move_package::source_package::layout::SourcePackageLayout;
 use std::{fmt::Display, fs::create_dir_all, io::Write, path::Path};
 
-// TODO get a stable path to this stdlib
-// pub const MOVE_STDLIB_PACKAGE_NAME: &str = "MoveStdlib";
-// pub const MOVE_STDLIB_PACKAGE_PATH: &str = "{ \
-//     git = \"https://github.com/move-language/move.git\", \
-//     subdir = \"language/move-stdlib\", rev = \"main\" \
-// }";
```

Comment:
Were these not used? If they were and are used, they don't appear anywhere here. Perhaps there's some more code that was not checked in, given that this PR is in draft mode :D

Reply:
They weren't used, my best guess is that this is leftover cruft from debugging or history. In particular, I think that since the move stdlib isn't mentioned in the generated toml anymore, it's not needed.